packages/nodes-base/nodes/Mcp/McpServer.ts

```diff
@@ -0,0 +1,199 @@
+import type { Tool } from '@langchain/core/tools';
+import { Server } from '@modelcontextprotocol/sdk/server/index.js';
+import type { RequestHandlerExtra } from '@modelcontextprotocol/sdk/shared/protocol.js';
+import type { JSONRPCMessage } from '@modelcontextprotocol/sdk/types.js';
+import {
+	JSONRPCMessageSchema,
+	ListToolsRequestSchema,
+	CallToolRequestSchema,
+} from '@modelcontextprotocol/sdk/types.js';
+import type * as express from 'express';
+import type { Logger } from 'n8n-workflow';
+import { zodToJsonSchema } from 'zod-to-json-schema';
```

Comment:
This module isn't available in `nodes-base` but we do have it in pnpm-workspace. Would make sense to use the workspace version both in `nodes-base` and in `@non/nodes-langchain`